src/components/map/MapSearch.js

```diff
@@ -107,37 +127,114 @@ const MapSearch = () => {
     let newState = [...activeFilters, filter];
     setActiveFilters(newState);
   };
-  return <div>
-
-<div className='map-menu'>
-      <Input  
-      className='map-search' 
-      placeholder="Search" 
-      allowClear onSearch={onSearch} 
-      suffix={suffix}
-      bordered={false}  />
-      <Divider style={{margin: '0px'}} />
+  
+  return (
+    <div className="map-menu">
+    
+
+
+      <Input
+        className="map-search"
+        placeholder="Search"
+        allowClear
+       
+        onChange={(e) => {
+          handleChange(e.target.value)
+          setIncidentsOfInterest(undefined)
+
+        }}
+        suffix={suffix}
+        
+      />
+
+
+
+      
+      
       <Collapse
-       style={{color: 'white'}}
+        style={{ color: 'white' }}
         defaultActiveKey={['1']}
-         onChange={callback}
+        onChange={callback}
+        bordered={false}
+        expandIconPosition="right"
+        ghost
+        accordion={true}
+      >
+        <Divider style={{ margin: '0px' }} />
+        <Panel
           bordered={false}
-          expandIconPosition='right'
-          ghost
-          >
+          style={{ color: 'white', padding: '0px' }}
+          header="More Info"
+          key="1"
+        >
           
-    <Panel bordered={false} 
-    style={{color: 'white', padding:'0px'}} 
-     header="More Info"
-      key="1"
-      >
-        <Divider style={{margin: '0px'}} />
-        <p>text</p>
-      
-    </Panel>
-    </Collapse>
+          <div className='incident-content'>
+            <List>
+              {!incidentsOfInterest? (
+                <div
+                className="incident-card"
+                
+                
+              >
+                <h2 className='incident-header'>{lastIncident?.title}</h2>
+                <h3 className='incident-location'> {lastIncident?.city}, {lastIncident?.state}</h3>
+                
+                <h3 className='incident-discription'>{lastIncident?.desc}</h3>
+                
+              </div>
+              ): (
+
+                !incidentsOfInterest ? filterDataList.map((data, index) => {
```

Comment:
I feel like we need to simplify this component. Perhaps parts can be extracted to separate components. Let's discuss.